sdk/python/foundation-models/llama2/promptFlow/standard-flow-sample/Fligh_fare/readme.md

```diff
@@ -0,0 +1 @@
+This is prompt flow sample for knowing flight fare with Llama-2-7b-chat.
```

Comment:
Add an small description about how this example works.

Reply:
Done.